components/apimgt/org.wso2.carbon.apimgt.rest.api.service.catalog/src/main/java/org/wso2/carbon/apimgt/rest/api/service/catalog/model/ExportArchive.java

```diff
@@ -0,0 +1,23 @@
+package org.wso2.carbon.apimgt.rest.api.service.catalog.model;
```

Comment:
missing License

Reply:
Added in 4208bf9c0f5c799e3876e177032977fb9c64e85f